shapely/geometry/base.py

```diff
@@ -164,9 +152,7 @@ class BaseGeometry(object):
     # _other_owned : bool
     #     True if this object's GEOS geometry is owned by another as in the
     #     case of a multipart geometry member.
-    __geom__ = EMPTY
     __p__ = None
-    _ndim = None
```

Comment:
I think that best all class attributes should be dropped in shapely 2.0. I read these as defaults for object-level attributes. These attributes add unnecessary state to the geometry objects.

As they are all private, I think they can be dropped. `_crs` and `_is_empty` could be replaced by properties calling `pygeos.get_srid` and `pygeos.is_empty` respectively if that is convenient for the implementation.

Reply:
Indeed, this needs further clean-up. I only removed the ones that I encountered explicitly while refactoring for now, but normally all others can be removed as well (they shouldn't be used anymore, or at least eventually they won't be used anymore). But so for sure, none of those needs to stay.